src/main/java/com/sekwah/advancedportals/bukkit/portals/Portal.java

```diff
@@ -513,12 +529,12 @@ public static boolean activate(Player player, AdvancedPortal portal, boolean doK
                 }, 20 * 10);
             }
 
-            if (portal.getDestiation() != null) {
+            if (portal.getDestinations().length != 0) {
                 if(plugin.isProxyPluginEnabled()) {
                     ByteArrayDataOutput outForList = ByteStreams.newDataOutput();
                     outForList.writeUTF(BungeeMessages.ENTER_PORTAL);
                     outForList.writeUTF(bungeeServer);
-                    outForList.writeUTF(portal.getDestiation());
+                    outForList.writeUTF(String.join(",", portal.getDestinations()));
```

Comment:
Id suggest only sending one location over rather than joining them. This would work better with backwards compatability.

Reply:
Yeah sorry, didn't really know what this method does since it was the last quick change to avoid using deprecated api

Reply:
No worries ^.^ thanks for the pr!